src/main/java/org/fundacionjala/at15/katas/pokerhands/saul/Constants.java

```diff
@@ -0,0 +1,13 @@
+package org.fundacionjala.at15.katas.pokerhands.saul;
+
+public final class Constants {
+    static final int ONE = 1;
```

Comment:
Need more context on the names of these constants. Maybe if we rename the class it would help, `Constants` sounds too generic